test/core/symbolFactoriesTests.ts

```diff
@@ -0,0 +1,99 @@
+///<reference path="../testReference.ts" />
+
+describe("SymbolFactory", () => {
+  describe("SymbolFactory generates correct path", () => {
+    let svg: d3.Selection<void>;
+    let generateSymbol: (type: string, size: number) => string;
+    let symbolSize = 5;
+
+    beforeEach(() => {
+      svg = TestMethods.generateSVG();
+      generateSymbol = (type: string, size: number) => d3.svg.symbol().type(type).size(size)(null);
+    });
+
+    it("returns a circle factory that generates circle based on symbolSize", () => {
+      let circleFoactory = Plottable.SymbolFactories.circle();
+      let actualSize = Math.PI * Math.pow(symbolSize / 2, 2);
+      let d = circleFoactory(5);
+      assert.strictEqual(d, generateSymbol("circle", actualSize), "a circle of set size is generated");
+      svg.append("path").data([{"d": d}]).attr("d", (d) => d.d).enter();
```

Comment:
When using `.enter()` we usually either add an element or similar, but instead it seems we don't do anything with the `enterSelection()`?